concourse/pipelines/pr_pipeline.yml

```
@@ -96,6 +101,9 @@ jobs:
             git submodule update --init --recursive
         inputs: [{ name: gpdb_pr }]
         outputs: [{ name: gpdb_src }]
+    - task: check_format
```

Should this be done in parallel?

---

No this should gate ICW because this step is much cheaper (~30s) than ICW. And we can fail fast here